app/Http/Requests/Pagination/PaginationRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests\Pagination;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class PaginationRequest extends FormRequest
+{
+    public function authorize(): bool
```

Comment:
You don't have to write this in every class, but create a base class, specify this function in it, and extends from it everywhere